static/app/views/dashboardsV2/manage/templateCard.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'sentry/components/button';
+import Card from 'sentry/components/card';
+import {IconAdd, IconGeneric} from 'sentry/icons';
+import overflowEllipsis from 'sentry/styles/overflowEllipsis';
+import space from 'sentry/styles/space';
+
+type Props = {
+  title: string;
+  widgetCount: number;
+};
+
+function TemplateCard({title, widgetCount}: Props) {
+  return (
+    <StyledCard>
+      <Header>
+        <IconGeneric size="48" />
+        <CardText>
+          <Title>{title}</Title>
+          <Detail>{widgetCount} Starter Widgets</Detail>
+        </CardText>
+      </Header>
+      <ButtonContainer>
+        <StyledButton priority="default">Preview</StyledButton>
+        <StyledButton priority="primary" icon={<IconAdd size="xs" isCircled />}>
```

Comment:
Minor css things:

I think in the design, these buttons have a gap between them and maybe the `Preview` button can have way less padding so that the "Add Dashboard" button has more space.